lr-cli/cortx_setup/commands/resource/discover.py

```diff
@@ -15,13 +15,47 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
-
+import time
 from ..command import Command
+from provisioner.commands import PillarSet
+from cortx.utils.discovery import Discovery
 
 
 class ResourceDiscover(Command):
+
+    def get_resource_map(self, resource_type='nodes[0]'):
+        retries = 30
```

Comment:
Can we make retries and sleep as configurable values at some common location so that we can tweak them whenever we run into issue (like as the resources grow in size, the time required to fetch resources will take longer)

Reply:
Done.
Added it in the config.py file.